fearless/Modules/Staking/StakingBondMore/View/StakingBondMoreViewLayout.swift

```diff
@@ -1,30 +1,45 @@
 import UIKit
 
 final class StakingBondMoreViewLayout: UIView {
+    private enum Constants {
+        static let hintIconWidth: CGFloat = 24.0
+    }
+
     let contentView: ScrollableContainerView = {
         let view = ScrollableContainerView()
         view.stackView.isLayoutMarginsRelativeArrangement = true
-        view.stackView.layoutMargins = UIEdgeInsets(top: 24.0, left: 0.0, bottom: 0.0, right: 0.0)
+        view.stackView.layoutMargins = UIEdgeInsets(top: UIConstants.hugeOffset, left: 0.0, bottom: 0.0, right: 0.0)
         view.stackView.spacing = UIConstants.bigOffset
         return view
     }()
 
-    let accountView: DetailsTriangularedView = UIFactory.default.createAccountView(for: .options, filled: true)
-    let collatorView: DetailsTriangularedView = UIFactory.default.createAccountView(for: .options, filled: true)
+    let accountView: DetailsTriangularedView = {
+        let view = UIFactory.default.createAccountView(for: .options, filled: true)
+        view.actionImage = nil
+        return view
+    }()
+
+    let collatorView: DetailsTriangularedView = {
+        let view = UIFactory.default.createAccountView(for: .options, filled: true)
+        view.actionImage = nil
+        return view
+    }()
 
     let amountInputView: AmountInputView = {
         let view = UIFactory().createAmountInputView(filled: false)
         return view
     }()
 
-    let networkFeeView = NetworkFeeView()
-
-    let actionButton: TriangularedButton = {
-        let button = TriangularedButton()
-        button.applyDefaultStyle()
-        return button
+    let hintView: IconDetailsView = {
+        let view = IconDetailsView()
+        view.iconWidth = Constants.hintIconWidth
+        view.imageView.contentMode = .top
```

Comment:
we doesn't have hint view for relaychain case , so it should be hidden by default and set visible only for parachain state